README.md

```diff
@@ -1,67 +1,39 @@
 # PHP MQTT Client
 
-phpMqttClient is a mqtt client library for PHP. Its based on the reactPHP socket-client and added the mqtt protocol specific functions. I hope its a better starting point that the existing php mqtt libraries. 
+phpMqttClient is an MQTT client library for PHP. Its based on the reactPHP
+socket-client and added the mqtt protocol specific functions. I hope its a
```

Comment:
I just split it on to two lines as it was running off the edge of my screen. I could undo the change if you prefer.